src/wm/keyboard.vala

```diff
@@ -143,6 +149,7 @@ namespace Budgie {
 
 		/* Reset InputSource list and produce something consumable by xkb */
 		void update_sources() {
+			sources = null;
```

Comment:
This looks superfluous - the same variable is immediately initialised afterwards

Reply:
The latter shouldn't be required either since it is being created on ln 64 now.

Reply:
I've fixed it in another way.